src/ActionsImporter/Program.cs

```diff
@@ -52,6 +52,22 @@
 var parsedArguments = parser.Parse(args);
 app.IsPrerelease = parsedArguments.HasOption(Common.Prerelease);
 
+var testCommandOnly = Environment.GetEnvironmentVariable("TEST_COMMAND_ONLY");
```

Comment:
> Also if there's somewhere else I can put this other than `Program.cs` I'd love to know, as that file is getting somewhat long and complex.

I do like this here since this piece of logic can potentially exit the application (versus exiting the app from a different file). Just my two cents. 🪙 🪙